plugins/org.eclipse.n4js.ui/src/org/eclipse/n4js/ui/wizard/workspace/WorkspaceWizardValidatorUtils.java

```diff
@@ -20,7 +20,7 @@
 public class WorkspaceWizardValidatorUtils {
 
 	private static final Pattern VALID_FOLDER_NAME_PATTERN = Pattern
-			.compile("[a-zA-z_](([a-zA-z_0-9-])|[a-zA-z_0-9-])*");
+			.compile("[a-zA-Z_.0-9-]+");
```

Comment:
I think '.' is a wildcard in this pattern. I'm not 100% certain, though.

Reply:
To save you guys some time (in order to add documentation):

```
Pattern p = Pattern.compile("[a-zA-Z_.0-9-]+");
p.matcher("abc.def").matches();  ==> true
p.matcher("abc!def").matches(); ==> false
```
So, it is not a wildcard here.